verilog/tools/ls/verilog-language-server_test.cc

```diff
@@ -1046,6 +1046,57 @@ TEST_F(VerilogLanguageServerSymbolTableTest,
   ASSERT_EQ(response_b["result"][0]["uri"], "file://" + module_a.filename());
 }
 
+TEST_F(VerilogLanguageServerSymbolTableTest, MultipleDefinitionsOfSameSymbol) {
+  const absl::string_view filelist_content = "bar_1.sv\nbar_2.sv\nfoo.sv";
+
+  const absl::string_view  //
```

Comment:
Why are the others `static constexpr` and this one just `const` ?

Reply:
Fixed